core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -120,6 +120,19 @@ pre {
   border: none;
 }
 
+.description-input {
+  overflow: hidden !important;
```

Comment:
Why do these all need "!important"? Are there other rules that will take precedence here?